### PR TITLE
Additional envs section

### PR DESCRIPTION
**Pull request by mlap**
Tried to keep this on the shorter side

### CODE REVIEW

File: manuscript/manuscript.Rmd [hunk 1]
```diff
@@ -439,17 +439,14 @@ Such examination makes it clear how the RL agent's use of a more dynamic strateg
...
-Our previous examples have focused on relatively simple examples that can also be addressed at least approximately by conventional methods, and also permit RL-based solutions to be found relatively efficiently on commodity hardware.
-To better illustrate the generality of RL approach and more typical complexity of conservation problems, we now introduce a third example for which we have not yet found an effective policy using RL or classical methods.
+Deep RL algorithms have proven successful on a range of problems that classical approaches could not solve despite immense computational resources -- e.g. deep RL can solve control problems with image-based observations that would be intractable with classical methods [@DQN]--; yet, getting deep RL to work in practice can be unexpectedly challenging and computationally intensive.
+Our previous examples have thus focused on relatively simple problems that can be addressed by conventional methods, and also permit RL-based solutions to be found relatively efficiently on commodity hardware.
```
Comment (cboettig) [inline manuscript/manuscript.Rmd:445]: I think we will get in trouble about our vague use of the term conventional methods.  I've mostly used this in a very narrow sense of SDP-based methods, which covers most of the actual examples in ecology.  We set up the second example with a non-autonomous change explicitly to rule out a direct SDP solution (technically our tipping point example is a partially-observed system, as you do not observe the value of the time-varying parameter moving the system towards the bifurcation).  I know our examples are still simple, but I think this contradicts our earlier statements about example 2, particularly as we don't have any optimal conventional solution to compare the results to.

File: manuscript/manuscript.Rmd [hunk 2]
```diff
@@ -439,17 +439,14 @@ Such examination makes it clear how the RL agent's use of a more dynamic strateg
...
-Nevertheless, deep RL approaches using neural network architectures have proven successful on problems that no amount of computational power would put into reach of purely conventional methods.
-Our previous examples have focused on relatively simple examples that can also be addressed at least approximately by conventional methods, and also permit RL-based solutions to be found relatively efficiently on commodity hardware.
-To better illustrate the generality of RL approach and more typical complexity of conservation problems, we now introduce a third example for which we have not yet found an effective policy using RL or classical methods.
+Deep RL algorithms have proven successful on a range of problems that classical approaches could not solve despite immense computational resources -- e.g. deep RL can solve control problems with image-based observations that would be intractable with classical methods [@DQN]--; yet, getting deep RL to work in practice can be unexpectedly challenging and computationally intensive.
```
Comment (cboettig) [inline manuscript/manuscript.Rmd:444]: I like where this is going, but the first sentence needs to a be a crisp 'topic sentence' for the paragraph, clearly stating the overall message of the paragraph.  There's really three sentences here.  We want to lead with the main point, something like:

> Ecology holds many open problems for deep RL.  It is straight-forward to extend the simple environments presented here to reflect greater biological complexity, or to consider a host of more realistic decision scenarios, from wildfires to epidemics. Training agents to discover viable policies for such scenarios will be much harder than it is in these introductory examples shown here. We provide an initial library of example environments at <https://boettiger-lab.github.io/conservation-gym>.

Comment (mlap) [reply manuscript/manuscript.Rmd:444]: This looks fine to me.

Comment (cboettig) [reply manuscript/manuscript.Rmd:444]: ok thanks, lets go with this.  I think it was my poor language in the previous version of the paragraph that was holding you up anyway, sometimes easier to start fresh!

**Pull request merged**